src/web/package.json

```
@@ -18,8 +18,7 @@
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "react-router-dom": "^6.27.0",
-    "react-router-hash-link": "^2.4.3",
-    "styled-components": "^6.1.13"
+    "styled-components": "^6.1.13",
     "react-router-hash-link": "^2.4.3"
```

nit: Should we sort these alphabetically? As this list grows, it'll make it easier to find things.
```suggestion
    "react-router-hash-link": "^2.4.3",
    "styled-components": "^6.1.13"
```

---

Yes, I can do that. I'll do this in the next pr